fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/FabricBiomes.java

```diff
@@ -1,37 +1,22 @@
-/*
- * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 package net.fabricmc.fabric.api.biomes.v1;
 
+import net.minecraft.util.registry.RegistryKey;
 import net.minecraft.world.biome.Biome;
 
-import net.fabricmc.fabric.impl.biome.InternalBiomeData;
+import net.fabricmc.fabric.impl.biome.FabricBiomesInternal;
 
-/**
- * General API that applies to all biome sources.
- */
 public final class FabricBiomes {
 	private FabricBiomes() { }
 
-	/**
-	 * Allows players to naturally spawn in this biome.
-	 *
-	 * @param biome a biome the player should be able to spawn in
-	 */
-	public static void addSpawnBiome(Biome biome) {
-		InternalBiomeData.addSpawnBiome(biome);
+	public static RegistryKey<Biome> register(RegistryKey<Biome> key, Biome biome) {
+		return FabricBiomesInternal.register(key, biome);
+	}
+
+	public static void addToOverworld(RegistryKey<Biome> key) {
```

Comment:
Hmm, since we broke api, maybe return `RegistryKey<Biome>` in these methods so we could chain these calls like a

```java
public static final RegistryKey<Biome> TEST_PLAINS = addToOverworld(FabricBiomes.register(RegistryKey.of(Registry.BIOME_KEY, new Identifier(MOD_ID, "test_plains"), DefaultBiomeCreator.createPlains(false)));
```

Yes it's long but it can allow for mod devs to make some helper methods in their registry classes.

Reply:
done 👍